src/extension.ts

```diff
@@ -55,38 +55,64 @@ async function fetchPackageInfo(requirement: PackageRequirement): Promise<Packag
     }
 }
 
-function presentPackageInfo(info: PackageInfo): string {
+function presentPackageInfo(info: PackageInfo): string[] {
     const summarySubPart: string = info.summary ? ` – ${linkify(info.summary, info.home_page)}` : ''
     const headPart: string = `**${linkify(info.name, info.package_url)}${summarySubPart}**`
     const emailSubpart: string = info.author_email ? ` (${info.author_email})` : ''
     const authorSubpart: string = info.author && info.author_email ? `By ${info.author}${emailSubpart}.` : ''
     const licenseSubpart: string = info.license ? ` ${info.license.replace(/ licen[cs]e/gi, '')} licensed.` : ''
     const versionPart: string = `Latest version: ${linkify(info.version, info.release_url)}.`
-    const infoPresentation: string = [
-        headPart, authorSubpart + licenseSubpart, versionPart
-    ].filter(Boolean).join('\n\n')
+    const versionPartLens: string = `Latest version: ${info.version}`
```

Comment:
This extra part seems redundant, it's the same as the previous one except for the link, but VS Code seems to be handling the link fine. Can we remove it and just rely on `versionPart`? Would simplify the code too

Reply:
Removed versionPart.